pages/about.html

```diff
@@ -1,12 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="/styles/style.css" rel="stylesheet">
 </head>
+
 <body>
-  
-</body>
-</html>
\ No newline at end of file
+  <header>
+    <h2>Kat's About Me</h2>
+  </header>
+  <main>
+    <nav>
+      <ul>
+        <li>
+          <a href='index.html'>Homepage</a>
+        </li>
+        <li>
+          <a href='portfolio.html'>Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+    <section>
+      <p id = "about">
```

Comment:
Based on W3 styling guide, it's better practice to not leave space before and after the `=`.

```html
      <p id="about">
```